en/docs/getting-started/overview.md

```diff
@@ -1,99 +1,101 @@
+<div style="width: 100%; overflow: hidden;">
+  
+<div class="rightContentOverview" style="width:40% !important;"> WSO2 API Manager is a fully open-source API management platform. It supports API designing, API publishing,  lifecycle management, application development, API security,  rate limiting, viewing statistics of APIs, as well as connecting APIs, API Products, and endpoints.
+  </div>
+  
+   <iframe width="450" height="260" src="https://www.youtube.com/embed/JejVjoaAc38" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+</div>
 
-## **What is the WSO2 API Manager** ?
 
-WSO2 API Manager is the only 100% open-source product that addresses all aspects of API management including API development, productization, consumption, security, rate limitation, monetization and analytics.
 
-![]({{base_path}}/assets/attachments/103327648/103346653.png)
-In brief, the WSO2 API Manager consists of 6 main components, the [API Publisher,](#api-publisher) the [API developer Portal,](#developer-portal) [API Gateway](#api-gateway) (now comes as a [API Microgateway](https://mg.docs.wso2.com/en/3.2.0/) as well ), [Key Manager,](#key-manager) [Traffic Manager](#traffic-manager) and the [API Analytics](#analytics). For more information on these components see [Basic Architecture and Key Components]().
+### Develop, Deploy and Manage APIs/API Products
 
-The key features of the product are listed below.
 
-## Key Features
+<div style="width: 100%; overflow: hidden;">
 
--  Design and Prototype APIs
--  Publish API Products and Govern the Use of APIs
--  Control Access and Enforce Security
--  Developer Portal
--  Manage Developer Community
--  Manage and Scale API Traffic
--  Monitor and Monetize
--  Pluggable, Extensible, and Themeable
--  Easily Deployable in Your Enterprise
--  WSO2 Platform Multi-Tenancy Support
--  Multiple Key Manager Support
+<div class="leftContentOverview" style="width:75% !important;">A well-designed API can make your APIs easy to use. WSO2 API Manager’s API Publisher guides you through API creation to API Publishing while adhering to the respective API’s specification. The API Publisher‘s user-friendly UI helps you create APIs, manage the API life cycle, secure your APIs, apply rate limiting policies, connect to endpoints, and publish your APIs easily. Additionally, you can create and publish API Products as well.
+  </div>
```

Comment:
Can we reduce the white space between the image and the text. This applies to all the images.
<img width="831" alt="Screenshot 2020-12-09 at 17 06 50" src="https://user-images.githubusercontent.com/5195851/101628681-2bafdf00-3a46-11eb-8553-1a94a12d4379.png">